README.md

```diff
@@ -2,14 +2,13 @@
 
 # Project Name
 
-> Description the project.
+Microverse Hello Project
 
```

Comment:
[OPTIONAL] 
- [ ] Kindly make sure your project has a detailed description telling what the project is about for it's a core standard for a repo to be documented professionally.